app/presenters/publishing_api_presenters/take_part.rb

```diff
@@ -0,0 +1,52 @@
+module PublishingApiPresenters
+  class TakePart
+    attr_reader :take_part_page, :update_type
+
+    def initialize(take_part_page, options = {})
```

Comment:
I agree we should be using them more but I'm going to leave them here as it's consistent with the other presenters.